### ISSUE TITLE
Improve the blog theme

### ISSUE THREAD

**Issue by davelab6** [sugarlabs/edit-fonts-activity#15]
Some nice ones I found in http://jekyllthemes.org....
- http://laobubu.net/jekyll-theme-EasyBook/
- https://alexcarpenter.github.io/material-jekyll-theme/

**Comment by davelab6** [comment]
Create and link profiles (https://github.com/sugarlabs/edit-fonts-activity/tree/gh-pages/profiles)

**Comment by eliheuer** [comment]
I was thinking about this today, will start on this

**Comment by YashAgarwal** [comment]
I like these.......

http://hemangsk.github.io/Gravity/index.html

https://github.com/tnguyen/blueface

https://alexcarpenter.github.io/material-jekyll-theme/

**Comment by davelab6** [comment]
I don't think we should go with Material... Gravity is too grey... Of those, I like blueface the best :)

**Comment by eliheuer** [comment]
Thoughts on this one if edited to fit the Sugar visual style? --  https://github.com/willianjusten/will-jekyll-template

**Comment by davelab6** [comment]
Yeah I like that :)

**Comment by YashAgarwal** [comment]
Its great :D even if we don't edit it
maybe we can remove the hamburger icon, the menu is quite useless for us

**Comment by YashAgarwal** [comment]
looks better in grey(rgb(51,51,51))

**Comment by eliheuer** [comment]
I've been trying to get a pull request setup with the [will-jekyll-template](https://github.com/willianjusten/will-jekyll-template), but I'm having trouble getting Gulp to compile my updated assets locally.  I'll keep trying to figure it out but if anyone has experience with Gulp I could use your help.

**Comment by YashAgarwal** [comment]
maybe you can try this https://robwise.github.io/blog/jekyll-and-gulp

**Comment by eliheuer** [comment]
Perfect, thank you!
